tests/images/Dockerfile.canary

```diff
@@ -7,11 +7,11 @@ RUN apt-get update && apt-get install -y curl \
     python-pip \
     vim \
     sudo
-
+    
 RUN pip install awscli
 
 # Install yq
-RUN sudo add-apt-repository ppa:rmescandon/yq && apt update && apt install -y yq
+RUN apt-get update && apt install -y apt-transport-https ca-certificates curl gnupg-agent software-properties-common && sudo add-apt-repository ppa:rmescandon/yq && apt update && apt install -y yq
```

Comment:
Why was this necessary? Can you update comment?

Reply:
In body of pull request - you need `software-properties-common` to use the `add-apt-repository` command.

Reply:
Ah sorry I missed. Weird that it is failing now.

Can you still update yq comment?

Reply:
ignore I saw outdated change